images/kcp-upgrade/upgrade.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Not really a "production" image, more related to tooling (as are the test images). I would probably move it out of the way. We might need to think about a slight refactor of the repository.

Reply:
Reading this comment, I feel we could actually refactor our repo to a great extent. For eg: ckcp and local can be placed in a new folder 'dev'. We could create 3 folders to store images - dev, test, automation. And place our images accordingly.

I will take this up sometime next week as it is outside the scope of this PR. Let me know what you think.

Reply:
Sounds good to me.